src/components/biggive-basic-card/biggive-basic-card.scss

```diff
@@ -31,18 +31,19 @@
       @include corner-clip-medium-bottom-left-top-right();
     }
   }
-  
+
   .icon {
     svg {
       width: 50px;
       height: auto;
     }
   }
-  h3 {
+  .title {
+    // these have to be the heading level passed in based on order of headings, but in terms of design we want it look like H3.
     @include heading-3();
```

Comment:
If we use a `headingLevel` that isn't 3, the component will still always `@include heading-3()`. Are you okay with this?

Reply:
Ok 👍

Reply:
Nice explanation.